presto-product-tests/bin/product-tests-suite-1.sh

```diff
@@ -2,7 +2,13 @@
 
 set -xeuo pipefail
 
+set +e
```

Comment:
I know I could have removed `e` from `set -xeuo pipefail` above, but I decided not to.
My thinking is that every script could begin with `set -xeuo pipefail` and then disable options it doesn't want.
This way `set +e` requirement is explicit rather than implied by the lack of `e`.

Reply:
Maybe add a comment explaining why we need it in this particular case

Reply:
heh, i realized this is wrong. we don't need it, because `||` "suspends" `set -e`'s effect